doc/source/simple-plugin.rst

```diff
@@ -30,89 +30,132 @@ to be able to run properly.  Any that are defined as optional need not necessari
 
 ::
 
+        _version = (1, 0, 0)
+        _required_framework_version = (2, 0, 0)
+
         @classmethod
         def get_requirements(cls):
-            return [requirements.TranslationLayerRequirement(name = 'primary',
-                                                             description = 'Memory layer for the kernel',
-                                                             architectures = ["Intel32", "Intel64"]),
-                    requirements.SymbolTableRequirement(name = "nt_symbols",
-                                                        description = "Windows kernel symbols"),
-                    requirements.PluginRequirement(name = 'pslist',
-                                                   plugin = pslist.PsList,
-                                                   version = (1, 0, 0)),
+            return [requirements.ModuleRequirement(name = 'kernel', description = 'Windows kernel',
+                                                   architectures = ["Intel32", "Intel64"]),
                     requirements.ListRequirement(name = 'pid',
                                                  element_type = int,
                                                  description = "Process IDs to include (all other processes are excluded)",
-                                                 optional = True)]
+                                                 optional = True),
+                    requirements.PluginRequirement(name = 'pslist',
+                                                   plugin = pslist.PsList,
+                                                   version = (1, 0, 0))]
```

Comment:
I think the minimum version of the `pslist` plugin should be `2.0.0`. Setting this to `1.0.0` will result in the error mentioned in the issue (#773).

Reply:
If unintended, this would be a new issue.

Reply:
```suggestion
                                                   version = (2, 0, 0))]
```